public/index.html

```diff
@@ -9,6 +9,11 @@
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
   <link rel="stylesheet" href="https://unpkg.com/tachyons/css/tachyons.min.css">
   <title>React App</title>
+  <style>
```

Comment:
This shouldn't be here, it would probably be better to apply it directly on a component.